source/module_basis/module_ao/parallel_orbitals.cpp

```diff
@@ -39,6 +39,11 @@ int Parallel_Orbitals::get_wfc_global_nbasis() const
 #endif
 }
 
+int Parallel_Orbitals::get_nbands() const
+{
+    return this->nbands;
```

Comment:
why do we need nbands from Parallel_Orbitals?

Reply:
`Parallel_Orbitals` has a function of `set_nloc_wfc_Eij` to set `this->ncol_bands`. `nbands` is needed in collect `psi_pool` to `psi` in parallel k diagonalization case. So `nbands` is needed in creating `psi_pool` with correct size. I add `nbands` in `Parallel_Orbitals` in order to create `psi_pool` in `hsolverLCAO` using its `paraV`, otherwise we need to call `GlobalV::NBANDS`, which is always avoided in new codes, as one knows.